### ISSUE TITLE
Unable to add note

### ISSUE THREAD

**Issue by 0xZ3RO** [0xZ3RO/ped#2]
Unable to add note. Program claims that the note exists when it doesn't.

![image.png](https://raw.githubusercontent.com/0xZ3RO/ped/main/files/6caef348-3c70-4e20-a093-8c20825e3672.png)

Delete another note first, and try adding another note. Program says that the note already exists. But when listing notes, there are no notes.

**Comment by Chongjx** [comment]
Hi @0xZ3RO We are unable to reproduce the bug on our side. Is there a note called "JavaDocs" that is archived? Archived note will not be displayed by default when using the list-n command. All notes, regardless or archive or unarchive, has to have unique title.

**Comment by 0xZ3RO** [comment]
> Hi @0xZ3RO We are unable to reproduce the bug on our side. Is there a note called "JavaDocs" that is archived? Archived note will not be displayed by default when using the list-n command. All notes, regardless or archive or unarchive, has to have unique title.

Yes there was a note called "JavaDocs" that was archived. Maybe make the error message mention that the note is archived, the user may think that the note is there but will be unable to find it using list-n.

**Comment by Chongjx** [comment]
Thanks for the clarification and suggestion!
